app/services/oneoffs/npq/data_cleanup/fix_applications_in_ecf_not_in_npq.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+module Oneoffs::NPQ::DataCleanup
+  class FixApplicationsInECFNotInNPQ
+    attr_reader :npq_application_ids
+
+    def initialize(npq_application_ids:)
+      @npq_application_ids = npq_application_ids
+    end
+
+    def run!(dry_run: true)
+      result = {}
+
+      ActiveRecord::Base.transaction do
+        result = npq_application_ids.each_with_object({}) do |npq_application_id, hash|
+          npq_application = NPQApplication.find_by(id: npq_application_id)
+          next unless npq_application
```

Comment:
I think we should raise an error here if the application id doesn't exist, it'll suggest the CSV data is incorrect

Reply:
Actually we need this because of line 36, we might delete it before we even get to it on the list.